exams/pearson.py

```diff
@@ -0,0 +1,147 @@
+"""
+Pearson specific exam code
+"""
+from functools import partial
+from operator import attrgetter
+import csv
+import logging
+
+from django.conf import settings
+import pycountry
+import pysftp
+
+from exams.exceptions import (
+    InvalidProfileDataException,
+    InvalidTsvRow,
+)
+
+# custom csv dialect for Pearson
+csv.register_dialect(
+    'pearsontsv',
+    delimiter='\t',
+)
+
+PEARSON_DATETIME_FORMAT = "%Y/%m/%d %H:%M:%S"
+
+log = logging.getLogger(__name__)
+
+
+def format_datetime(dt):
+    """
+    Formats a datetime to Pearson's required format
```

Comment:
Can you add the argument and return value using the sphinx-napoleon syntax? We generally follow the edX styleguide for that: http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#docstrings